README.md

```diff
@@ -96,6 +96,37 @@ You can also set the following mount points by passing the `-v /host:/container`
 directory has the appropriate permissions and that the owner and group of the directory
 matches the user UID or name which is running inside the container.**
 
+Inclusion of the `oc` binary
+---------------------------------
+
+To assist in interacting with the OpenShift API server while using this image, the `oc` binary, the CLI command for OpenShift, has been
+installed in this image.
+
+However, it needs to noted that backward compatibility is not guaranteed between different versions of `oc` and the OpenShift
+API Server.  As such, it is recommended that you align versions of this image present in the nodes of your cluster with your 
+OpenShift API server.  In other words, you should use the version specific tag instead of the `latest` tag.
+
+|  Jenkins image version      | `oc` client version      |
+| :-------------------------- | ------------------------ |
+|  `jenkins-*-centos7:v3.7`   | 3.7 `oc` binary          |
+|  `jenkins-*-centos7:v3.6`   | 3.6 `oc` binary          |
+|  `jenkins-*-centos7:latest` | 3.9 `oc` binary          |
+|  `jenkins-*-rhel7:v3.7`     | 3.7 `oc` binary          |
+|  `jenkins-*-rhel7:v3.6`     | 3.6 `oc` binary          |
+|  `jenkins-*-rhel7:latest`   | 3.6 `oc` binary          |
```

Comment:
Might be worth reversing the two paragraphs that follow and putting an asterisk here to link the following paragraph more closely

Reply:
I imagine there is some trick with markdown language to employ the asterisk like you are suggesting .... unless you know different, I'll start hunting for examples / doc references